.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,16 @@ Closes #<issue number here>.
 
 ### Description of changes
 
-<!--What code changes did you make? Have you made any important design decisions?-->
+<!--
+What code changes did you make? 
+Have you made any important design decisions?
+What AWS Service feature use cases are you adding (describe if omitting) ?
```

Comment:
This line is a little confusion to me. 

```suggestion
What AWS use cases does this change enable? To enable the use cases, which AWS service features are utilized?
```

What does `describe if omitting` mean? Does it mean if the contribution does not answer the question, they need to tell why?